util/disk-image-validator/test_gem5_bridge.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
Using /usr/bin/env is the safest way to do this

```suggestion
#!/usr/bin/env bash
```